api/src/pcapi/routes/backoffice/templates/components/search/result.html

```diff
@@ -19,7 +19,7 @@
     </p>
     <div>
       <div class="row row-cols-sm-1 row-cols-md-2 row-cols-lg-3 g-4">
-        {% for row in rows.items %}<div class="col">{{ result_card_switch(row, result_type) }}</div>{% endfor %}
+        {% for row in rows.items %}<div class="col">{{ result_card_switch(row,loop.index,terms, result_type) }}</div>{% endfor %}
```

Comment:
Je pense que je suis passé à côté mais je vois pas d'où vient ce `loop` ?

Reply:
C'est un built-in de jinja dans une boucle for. Ça permet d'avoir l'index de la boucle. Ce qui permet de définir un rang de recherche.

Reply:
Ah ok je connaissais pas ^^